nre/bpf/nodefw/nodefw-ebpf/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "nodefw-ebpf"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+aya-bpf = { git = "https://github.com/aya-rs/aya" }
+aya-log-ebpf = { git = "https://github.com/aya-rs/aya" }
+nodefw-common = { path = "../nodefw-common" }
+network-types = "0.0.5"
+memoffset = "0.9"
+
+[[bin]]
+name = "nodefw"
+path = "src/main.rs"
+
+[profile.dev]
+opt-level = 3
+debug = false
+debug-assertions = false
+overflow-checks = false
+lto = true
+panic = "abort"
+incremental = false
+codegen-units = 1
+rpath = false
+
+[profile.release]
+lto = true
+panic = "abort"
+codegen-units = 1
+
+[workspace]
+members = []
```

Comment:
Thisis the actual bpf module right? this can and likely should remain as its own workspace

Reply:
yep it is, taking that workspace section away is doom for the build.